_episodes/10-conflicts.md

```diff
@@ -211,53 +295,59 @@ To configure Git to avoid creating backups at all:
-$ git commit
+$ git merge -s recursive -Xours less-avocados  # merge and in doubt take the changes from current branch
+```
+
+Or:
+
+```shell
+$ git merge -s recursive -Xtheirs less-avocados  # merge and in doubt take the changes from less-avocados branch
 ```
 
 ---
 
 ## Aborting a conflicting merge
 
-- Imagine it is Friday evening, you try to merge but have conflicts all over
-  the place.
-- You do not feel like resolving it now and want to undo the half-finished
-  merge.
-- Or it is a conflict that you cannot resolve and only your colleague knows
-  which version is the one to keep.
-- There is no reason to panic and delete the whole repository.
-- Simply undo the commit and go home.
-
-```shell
-$ git merge --abort  # undo the merge step
-```
+- Imagine it is Friday evening, you try to merge but have conflicts all over the place.
+- You do not feel like resolving it now and want to undo the half-finished merge.
+- Or it is a conflict that you cannot resolve and only your colleague knows which version is the one to keep.
 
-The repository looks exactly as it was before the merge.
+What to do?
 
----
+- There is no reason to delete the whole repository.
+- You can undo the broken merge by resetting the repository to `HEAD` (last committed state).
 
-## Exercise: Resolve the conflict 
+```shell
+$ git merge --abort
+```
 
-- Use one of the methods described above to resolve the cilantro conflict
+The repository looks then exactly as it was before the merge.
 
 ---
 
 ## Avoiding conflicts
 
-- Conflicts can be avoided if you think and talk with your colleagues before committing.
-- Think and plan to which branch you will commit to.
-- Fortran people: modifying common blocks often causes conflicts.
-- Modifying global data often causes conflicts.
-- Monolithic entangled spaghetti-code maximizes risk of conflicts.
-- Modular programming minimizes risk of conflicts.
-- Ball-of-mud branches for "everything" maximize risk of conflicts.
-- One branch for one task only.
-- Resolve conflicts early.
-- If the branch affects code that is likely to be modified by others:
-  - the branch should be short-lived and/or merge often to the main development line
-  - the branch should merge the main development line often to stay up-to-date
+- Human measures
+  - Think and plan to which branch you will commit to.
+  - Ball-of-mud branches for "everything" maximize risk of conflicts.
+  - One branch for one task only.
+- Collaboration measures
+  - Conflicts can be avoided if you think and talk with your colleagues before committing.
+  - Semantic conflicts that merge but don't work: Importance of talking!
+- Project layout measures
+  - Fortran people: modifying common blocks often causes conflicts.
+  - Modifying global data often causes conflicts.
+  - Monolithic entangled spaghetti-code maximizes risk of conflicts.
+  - Modular programming minimizes risk of conflicts.
+- Technical measures
```

Comment:
Oh, what about keeping up to date with upstream / pulling often?  I have often begun working, and pull when I am about to push with predictable results...  This should also be bold.